utils/cliutils/commandsflags.go

```diff
@@ -1186,6 +1189,34 @@ var flagsMap = map[string]cli.Flag{
 		Name:  xrOutput,
 		Usage: "[Default: table] Defines the output format of the command. Acceptable values are: table and json.` `",
 	},
+	Mvn: cli.StringFlag{
+		Name:  Mvn,
+		Usage: "[Optional] Request audit for Maven project.` `",
```

Comment:
...for Maven project. --> ...for a Maven project

Let's make the same change (a addition) for the below new flags.